build_tools/ci/cpu_comparison/run.py

```diff
@@ -473,6 +472,59 @@ def _execute(self, config):
         return self.vs_cpu(config)
 
 
+class Matmul4d(BaseMatmul):
+    """
+    A test of linalg.generic with 4d inputs and output implementing form:
+    C += matmul4d(A,B) where A:MxKxM0xK0, B:NxKxK0xN0, C:NxMxM0xN0
+
+    Note that the order of the input dims for this operation corresponds to
+    the L2 shapes of a standard matmul op after the first level packing.
```

Comment:
You have to be intimately familiar with the current pipeline to know what this sentence means, can you provide a more high-level description in terms of transposition? 

Actually is this sentence even needed, doesn't `A:MxKxM0xK0, B:NxKxK0xN0, C:NxMxM0xN0` describe the computation exactly (I assume that anyone reading this will know that K and K0 are reduction dimensions).

Reply:
Okay, updated the documentation.